contracts/TokenSale.sol

```diff
@@ -197,13 +197,13 @@ contract TokenSale is FinalizableCrowdsale, Pausable {
         uint256 starAllocationToTokenSale = starToken.allowance(beneficiary, this);
         if (starAllocationToTokenSale > 0) {
             // calculate token amount to be created
-            uint256 tokens = starAllocationToTokenSale.mul(starRate);
+            uint256 tokens = starAllocationToTokenSale.mul(starRate).div(1000);
```

Comment:
Why 1000?

Reply:
Should be 100

Reply:
Doesn't matter what we insert here as long as the passed `starRate` is accordingly. Maybe I should add a comment to the init function@_starRate param.

Reply:
1000 allows us to accept a bit lower `starRates` than just 100.